specification/security/resource-manager/Microsoft.Security/preview/2022-01-01-preview/ServerVulnerabilityAssessmentsSettings.json

```diff
@@ -0,0 +1,328 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2022-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/serverVulnerabilityAssessmentsSettings": {
```

Comment:
>serverVulnerabilityAssessmentsSettings [](http://example.com/codeflow?start=66&length=38)

I would wonder to the PMs etc if we should nitpick on the pluralization.
Should it be the assessments which are plural, or just the settings?


serverVulnerabilityAssessmentsSettings

vs

serverVulnerabilityAssessmentSettings

Reply:
(PS: I partly just figure its going to be harder to say correctly and remember to type the extra 'S' in the middle)